website/source/docs/job-specification/service.html.md

```diff
@@ -151,6 +151,13 @@ Connect][connect] integration.
 - `meta` <code>([Meta][]: nil)</code> - Specifies a key-value map that annotates
   the Consul service with user-defined metadata.
 
+- `canary_meta` <code>([Meta][]: nil)</code> - Specifies a key-value map that
+  annotates the Consul service with user-defined metadata when the service is
+  part of an allocation that is currently a canary. Once the canary is
+  promoted, the registered meta will be updated to those specified in the
+  `meta` parameter. If this is not supploed, the registered meta will be set to
```

Comment:
typo here: "supplied"

If I understand this correctly, the promoted canary's metadata is removed if the `meta` stanza is default (`nil`). Maybe we should specifically call this out?